From a582ca642f4817dd02e65a3ecc55e951008969b2 Mon Sep 17 00:00:00 2001
From: Jim Meyering <meyering@redhat.com>
Date: Sat, 26 Jun 2010 09:22:59 +0200
Subject: [PATCH] tests: adjust sun-partition-creating test to conform

* tests/t4000-sun-raid-type.sh: Adjust partition size so the
end falls on a cylinder boundary.
---
 tests/t4000-sun-raid-type.sh |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

Index: b/tests/t4000-sun-raid-type.sh
===================================================================
--- a/tests/t4000-sun-raid-type.sh
+++ b/tests/t4000-sun-raid-type.sh
@@ -26,7 +26,7 @@
 
 N=2000 # number of sectors
 dev=sun-disk-file
-exp="BYT;\n---:${N}s:file:$ss:$ss:sun:;\n1:0s:50s:51s"
+exp="BYT;\n---:${N}s:file:$ss:$ss:sun:;\n1:0s:127s:128s"
 test_expect_success \
     'create an empty file as a test disk' \
     'dd if=/dev/zero of=$dev bs=${ss}c count=$N 2> /dev/null'
@@ -38,7 +38,7 @@
 
 test_expect_success \
     'create a single partition' \
-    'parted -s $dev unit s mkpart ext2 0s 50s > out 2>&1'
+    'parted -s $dev unit s mkpart ext2 0s 127s > out 2>&1'
 test_expect_success 'check for empty output' 'compare out /dev/null'
 
 test_expect_success \
